apache-couchdb-mixin/README.md

```diff
@@ -53,8 +53,8 @@ scrape_configs:
       - targets: [localhost]
         labels:
           job: integrations/apache-couchdb
-          instance: '<your-instance-name>'
-          cluster: '<your-cluster-name>'
+          instance: "<your-instance-name>"
+          cluster: "<your-cluster-name>"
           __path__: /var/log/couchdb/couchdb.log
```

Comment:
Is this supposed to be changed? like does this mixin affect yours?

Reply:
This was not intentional, I think it was just an auto-format on save that happened when copying this other file